deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -209,6 +209,8 @@ customDashboards:
     enabled: true
   LocalPV:
     enabled: true
+  lvm-LocalPV:
```

Comment:
LVMLocalPV or lvmLocalPV

Reply:
I think in the mixin PR, we have decided to follow the camel-case approach.

Reply:
Yes @avishnu . We have decided to follow the camel-case approach for configuration. 
This is the directory name that we use to store dashboards json.  We can rename the directory to lvmLocalPV.